### PR TITLE
tests redone

### PR DESCRIPTION
**Pull request by LadPaule**
## Description
Please include a summary of the change and relevant motivation and context. 
List any dependencies that are required for this change.

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(issue number 1) #(issue number 2 if applicable)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [x] I have added tests that prove my fix is effective and that this feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/services/produce_api/produceAPI/produce/tests.py [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
+
+    def test_updateProduct(self):
+        data = {"name": "testchange"}
+        response = self.client.put('/products/2/', data)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+class CredentialsTestCase(APITestCase):
+    test_url = reverse('prod')
+    def setup(self):
+        self.user = User.objects.create_user(username='testuser', email='testuser@farmrail.com', password='verystrongpsw')
```
Comment (amkayondo) [inline src/services/produce_api/produceAPI/produce/tests.py:30]: refactor line by placing valuables on new lines

**Review by amkayondo — CHANGES_REQUESTED** [review]
add the requested changes and good work

File: package.json [hunk 2]
```diff
@@ -1,49 +1,53 @@
 {
```
Comment (kmwamasali) [inline package.json]: remove all edits to the package.json file, all folder dependencies should be added at folder level and not the global scope

Comment (kmwamasali) [reply package.json]: Create the pckg.json in your folder and work with it from there

File: src/services/produce_api/produceAPI/produce/tests.py (hunk 1 above)
Comment (kmwamasali) [reply src/services/produce_api/produceAPI/produce/tests.py:30]: not applicable in python

**Review by kmwamasali — CHANGES_REQUESTED** [review]

**Pull request closed without merging**